apps/student-app/src/app/screens/game-screen.tsx

```diff
@@ -3,32 +3,38 @@ import { useTimer } from 'react-timer-hook';
 
 const GameScreen = ({
   currentWord,
-  correctAction,
-  skippedAction,
+  onCorrectAction,
+  onSkippedAction,
+  onExpiry,
   score,
-  countdown
 }: {
   currentWord: Word;
-  correctAction: () => void;
-  skippedAction: () => void;
+  onCorrectAction: () => void;
+  onSkippedAction: () => void;
```

Comment:
just renamed these to prepend "on" which is more the react way. It is something I recently learned.

Reply:
Yeah, makes sense.
done.